macosx/InfoActivityViewController.mm

```diff
@@ -99,10 +99,9 @@ - (NSRect)viewRect
     NSRect viewRect = self.view.frame;
 
     CGFloat difference = self.fHeightChange;
-    if (difference != 0)
+    if (difference)
     {
         viewRect.size.height -= difference;
```

Comment:
The same comment as before: why do we need a check for zero to subtract? It's not like subtracting zero requires massive resources and takes half an hour to complete.

Reply:
I think at one point without the check it would lead to a hang,
this doesn't seem to be the case anymore